### ISSUE TITLE
Make pivot structure query-able 

### ISSUE THREAD

**Issue by michaeljosephrosenthal** [nnajm/orb#30]
Would be nice to get the json representation of the current layout, with rows, columns, etc.
Ultimately wanting to set a cookie for reloading the user's desired layout

**Comment by maartendraijer** [comment]
Yeah, something like this would be very handy: http://nicolas.kruchten.com/pivottable/examples/onrefresh.html.
